homeworks/serhii.ripchanskyi_Ripchanskiy/11-dependency-injection/vechicle/src/app/app.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component } from '@angular/core';
+import { finalize } from 'rxjs/operators';
+
+import { Vehicle } from './models/';
+import { VehicleService } from './services';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
```

Comment:
Sorry, you forgot to implement the `OnInit` hook. This is important part to `ngOnInit` work 

You code should looks like this:
`export class AppComponent implements OnInit`

Reply:
Please, fix this moment and I can merge your MR :+1:  :pray: